crypto/err/err.c

```diff
@@ -839,17 +721,7 @@ const char *ERR_reason_error_string(unsigned long e)
 
 void ERR_remove_thread_state(const CRYPTO_THREADID *id)
```

Comment:
I'm not sure we can do this API any more. We can't remove the thread state for an arbitrary thread id...can we?